crates/tools/gnullvm/readme.md

```diff
@@ -0,0 +1,8 @@
+The Windows umbrella lib (targeting mingw-w64 LLVM tooling) is generated using the following steps:
+
+0. Ensure MSYS2 MinGW environment is installed (https://www.mingw-w64.org/downloads/)
+1. Execute: `C:\msys64\clang64.exe`
+2. Execute: `pacman -Syuu --noconfirm` (repeat until no further updates available)
+3. Execute `pacman --needed -S mingw-w64-clang-x86_64-toolchain`
+4. Navigate to crate root
+5. Execute: `PATH=$(cygpath -u $USERPROFILE)/.cargo/bin:$PATH cargo run -p tool_gnullvm --target x86_64-pc-windows-gnu -- all`
```

Comment:
Tweaked this line to convert path with `cygpath` because MSYS2 doesn't understand Win32 style path from `$USERPROFILE`:
```
$ PATH=$USERPROFILE/.cargo/bin:$PATH

$ which cargo
which: no cargo in (C:\Users\mateusz/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/opt/bin:/c/Windows/System32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0/:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)

$ echo $PATH
C:\Users\mateusz/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/opt/bin:/c/Windows/System32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0/:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl

$ PATH=$(cygpath -u $USERPROFILE)/.cargo/bin:$PATH

$ which cargo
/c/Users/mateusz/.cargo/bin/cargo

$ echo $PATH
/c/Users/mateusz/.cargo/bin:C:\Users\mateusz/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/opt/bin:/c/Windows/System32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0/:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
```